docs/reference/rollup/apis/rollup-api.asciidoc

```diff
@@ -0,0 +1,191 @@
+[role="xpack"]
+[testenv="basic"]
+[[rollup-api]]
+=== Rollup API
+++++
+<titleabbrev>Rollup</titleabbrev>
+++++
+
+Aggregates an index's time-series data and stores the results in another index.
+For example, you can roll up hourly data into daily or weekly summaries.
+
+[source,console]
+----
+POST /my-index-000001/_rollup
+{
+  "rollup_index": "my-rollup-index",
+  "groups": {
+    "date_histogram": {
+      "field": "@timestamp",
+      "calendar_interval": "1d"
+    },
+    "terms": {
+      "fields": [
+        "my-keyword-field",
+        "my-other-keyword-field"
+      ]
+    }
+  },
+  "metrics": [
+    {
+      "field": "my-numeric-field",
+      "metrics": [
+        "min",
+        "max",
+        "avg"
+      ]
+    }
+  ]
+}
+----
+// TEST[setup:my_index]
+
+[[rollup-api-request]]
+==== {api-request-title}
+
+`PUT /<index>/_rollup`
+
+[[rollup-api-prereqs]]
+==== {api-prereq-title}
+
+If the {es} {security-features} are enabled, you must have the `manage` cluster
+privilege to use this API. See <<security-privileges>>.
```

Comment:
Note for reviewers: The `manage_rollup` privilege doesn't seem to have any affect on this API. If this is intentional, I'll update the privileges docs in another PR.

Reply:
I think you found a bug :)

Reply:
I'll open up an issue for that. Thanks, Tal!

Reply:
opened #65469. should fix this

Reply:
Awesome! Saved me a step. Thanks.